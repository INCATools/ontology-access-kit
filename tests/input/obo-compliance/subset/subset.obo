format-version: 1.4
subsetdef: S "my subset"
ontology: subset

[Term]
id: X:1
subset: S

