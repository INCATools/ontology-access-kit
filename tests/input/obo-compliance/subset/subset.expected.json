{
  "graphs" : [ {
    "id" : "http://purl.obolibrary.org/obo/subset.owl",
    "meta" : {
      "basicPropertyValues" : [ {
        "pred" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
        "val" : "1.4"
      } ]
    },
    "nodes" : [ {
      "id" : "http://purl.obolibrary.org/obo/X_1",
      "type" : "CLASS",
      "meta" : {
        "subsets" : [ "http://purl.obolibrary.org/obo/subset#S" ]
      }
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#SubsetProperty",
      "lbl" : "subset_property",
      "type" : "PROPERTY"
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
      "lbl" : "has_obo_format_version",
      "type" : "PROPERTY"
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#id",
      "lbl" : "id",
      "type" : "PROPERTY"
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#inSubset",
      "lbl" : "in_subset",
      "type" : "PROPERTY"
    } ]
  } ]
}